extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DevServicesDatasourceResultBuildItem.java

```diff
@@ -1,55 +0,0 @@
-package io.quarkus.datasource.deployment.spi;
-
-import java.util.Collections;
-import java.util.Map;
-import java.util.Optional;
-
-import io.quarkus.builder.item.SimpleBuildItem;
-import io.quarkus.datasource.common.runtime.DataSourceUtil;
-
-public final class DevServicesDatasourceResultBuildItem extends SimpleBuildItem {
```

Comment:
Is it ok to remove this class completely? I mean it's public and part of the `spi` package so I'd consider it to be part of the public API...

Reply:
You are right on that. There was no usage on quarkus repo and external extensions but I found at least 2 usages on github.

Reply:
As @mkouba says, we cant remove build items that are part of the SPI